packages/cmf/src/settings.js

```diff
@@ -36,6 +49,11 @@ export function nonMemoized(state, ownProps, componentName, componentId) {
 
 	if (viewId && state.cmf.settings.props[viewId]) {
 		viewProps = state.cmf.settings.props[viewId] || {};
+	} else if (componentName && componentName.indexOf('(') !== -1) {
```

Comment:
I found that more readable:

`if (componentName && componentName.includes('(')) {`